frontend-react/src/hooks/UseFileHandler.ts

```diff
@@ -155,7 +155,7 @@ function calculateFileSelectedState(
         };
     } catch (err: any) {
         // todo: have central error reporting mechanism.
```

Comment:
@kevinhaube Are `warn`s still tracked in this app?  I remember you mentioned in a meeting last week that we'd likely want to continue escalating JavaScript errors (which I agree with) so I'm curious if there have there been any thoughts about another third-party integration like Sentry or Bugsnag.  I definitely get that we don't want to be alerting whoever's on call through PagerDuty for potentially small/inconsequential errors, but I think it'd still be useful to be able to continue to track errors to determine if, say, the most recent release has a critical bug.  It could also be good for backlogging lower-priority bugs in the future, too!

BUT this could be a point relevant to the app insights discussion -- we might be able to have better session tracing with that!  I haven't dug into the options we have there yet, though 😅

Reply:
Gotcha, that sounds great to me.  Thanks for following up here! 🙏